db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+comedy = Category.create!(name: 'Comedy')
+drama = Category.create!(name: 'Drama')
+
+Video.create!(title: 'Family Guy', description: 'Brian makes Stewie watch viral video.', small_cover_url: '/tmp/family_guy.jpg', category: comedy)
```

Comment:
Slightly better way how to write this would be a different format for your hash like this:

```ruby
Video.create!(title: "...",
              description: "...",
              small_cover_url: "...",
              another_url: "...")
```